samples/bluetooth/direction_finding_connectionless_tx/src/main.c

```diff
@@ -136,7 +137,14 @@ void main(void)
 	}
 	printk("success\n");
 
-	bt_le_ext_adv_oob_get_local(adv_set, &oob_local);
+	printk("Get extended advertising address...");
```

Comment:
Am I missing something here? What is the relation between advertising address and the `oob_local`?

Reply:
@Emil-Gydesen-Bose The Advertising address is part of the OOB information.

Reply:
Ah, I had accidentally looked at `struct bt_le_oob_sc_data` :)